benchmarks/multiplatform/benchmarks/src/desktopMain/kotlin/Benchmarks.desktop.kt

```diff
@@ -0,0 +1,21 @@
+import java.io.File
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+actual fun saveBenchmarksOnDisk(name: String, stats: BenchmarkStats) {
```

Comment:
Have not you tried [kotlinx.io](https://github.com/Kotlin/kotlinx-io) to make it common?

Reply:
Checked native macOs - it saves files